src/components/BookList.js

```diff
@@ -0,0 +1,21 @@
+import Book from './Book'
+
+function BookList({ books }) {
+  return (
+    <ul>
```

Comment:
Don't you want to remove the list styles? Or are they kind or removed when adding a flexbox to the `li`? 🤔

Reply:
Stefan and I just looked at the code again and it is indeed the case that `display: flex` overrides `display: list-item`. This removes the default values for the list.

Reply:
Nice! Learned something new 🙂